src/middlewared/middlewared/plugins/rdma/rdma.py

```diff
@@ -133,3 +134,12 @@ def get_card_choices(self):
             names = [link['rdma'] for link in v['links']]
             v['name'] = ':'.join(sorted(names))
         return list(grouper.values())
+
+    @private
+    async def capable_services(self):
+        result = []
+        if await self.middleware.call('system.is_enterprise'):
```

Comment:
We don't want this to be available for our mini based hardware platforms. Furthermore `system.is_enterprise` calls `system.license`.

We can do something like this
```
is_ent = await self.middleware.call('system.is_enterprise')
if is_ent and 'MINI' not in await self.middleware.call('truenas.get_chassis_hardware'):
    if await self.middleware.call('rdma.get_link_choices', True):
        result.append(...
        
return result